dataservices/dataservices.go

```diff
@@ -0,0 +1,239 @@
+package main
+
+import (
+	"net/http"
+	"strings"
+
+	"github.com/tidepool-org/platform/data"
+	"github.com/tidepool-org/platform/logger"
+	"github.com/tidepool-org/platform/store"
+	"github.com/tidepool-org/platform/user"
+	"github.com/tidepool-org/platform/version"
+
+	"github.com/tidepool-org/platform/Godeps/_workspace/src/github.com/ant0ine/go-json-rest/rest"
+)
+
+const (
+	missingPermissionsError = "missing required permissions"
```

Comment:
We might want to refactor out all error message constants into one file for each package.  This will make it easy to use in the API docs.